source/crud/tests/unified/create-null-ids.yml

```diff
@@ -95,75 +95,3 @@ tests:
         object: *collection
         arguments: {filter: *null_id_filter}
         expectResult: 1
-
-  - description: inserting _id with type undefined via insertOne
```

Comment:
@nbbeeken: I assume these failures were missed in the original PR because Node silently converts undefined to null. In any event, undefined was never in the original scope of [DRIVERS-2124](https://jira.mongodb.org/browse/DRIVERS-2124) so let's just forget about this.

Reply:
Yep, exactly right, good catch thanks for fixing this so quick!